src/dbus_api/api.rs

```diff
@@ -65,9 +65,9 @@ fn get_next_arg<'a, T>(iter: &mut Iter<'a>, loc: u16) -> Result<T, MethodErr>
 }
 
 /// Get filesystem name from object path
-fn object_path_to_pair(dbus_context: &DbusContext,
-                       fs_object_path: &dbus::Path)
-                       -> Result<(Uuid, Uuid), (MessageItem, MessageItem)> {
+fn fs_object_path_to_pair(dbus_context: &DbusContext,
+                          fs_object_path: &dbus::Path)
+                          -> Result<(dbus::Path<'static>, Uuid), (MessageItem, MessageItem)> {
```

Comment:
I was able to get rid of the method entirely in a later commit, so the outdated header comment is pretty much moot, and very hard to fix.

I don't think it would help. It's what the dbus-rs developer has chosen to use to represent the data view of an ObjectPath. It's hard to put that in a type. If you want to distinguish it from a standard library Path, the code forces use to do that with the ```dbus``` prefix.